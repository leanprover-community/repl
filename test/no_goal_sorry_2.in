{"cmd" : "example : True := by\n  sorry\n  sorry", "sorries": "individual"}
