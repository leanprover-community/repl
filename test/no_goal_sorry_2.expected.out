{"sorries":
 [{"proofState": 0,
   "pos": {"line": 2, "column": 2},
   "goal": "‚ä¢ True",
   "endPos": {"line": 2, "column": 7}}],
 "messages":
 [{"severity": "error",
   "pos": {"line": 3, "column": 2},
   "endPos": {"line": 3, "column": 7},
   "data": "No goals to be solved"}],
 "env": 0}

